custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -129,6 +128,12 @@ def get_person_case_structure(self):
             },
         }
 
+        if self._location:
```

Comment:
Kind of. It is a "[weird owner id](https://github.com/dimagi/commcare-hq/blob/db17403a17c815ed663943e90afca3d5032d2c07/corehq/apps/users/util.py#L19-L26)" so we don't do stuff in the restore with those IDs. 
We are using it elsewhere in the eNikshay app to signify "a closed case that we might want to open in the app after searching for it with case claim at some point".